listenbrainz/listen.py

```diff
@@ -179,6 +179,8 @@ def to_influx(self, measurement):
             'time' : self.ts_since_epoch,
             'tags' : {
                 'user_name' : escape(self.user_name),
+                'artist_msid' : self.artist_msid,
```

Comment:
Question: do tags not get returned when we do `select count(*) from measurement`? Asking this because I removed them both from fields and added into tags, and the listen count test [here](https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/listenstore/tests/test_influxlistenstore.py#L197) started to fail because there wasn't a `count_recording_msid` field in the results of the query.